x-pack/plugins/data_usage/server/routes/internal/data_streams_handler.ts

```diff
@@ -41,12 +25,15 @@ export const getDataStreamsHandler = (
         core.elasticsearch.client.asSecondaryAuthUser
       );
 
-      const body = meteringStats
-        .sort((a, b) => b.size_in_bytes - a.size_in_bytes)
-        .map((stat) => ({
-          name: stat.name,
-          storageSizeBytes: stat.size_in_bytes ?? 0,
-        }));
+      const body =
+        meteringStats && !!meteringStats.length
```

Comment:
@neptunian I added this change just in case the metering stats API does not return anything useful.